.changeset/famous-dancers-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': major
```

Comment:
Why is this a `major`?

Reply:
I don't know, after running `pnpm exec changeset`, `major` is the only option left